Kudu.Contracts/SiteExtensions/ISiteExtensionManager.cs

```diff
@@ -17,7 +17,7 @@ public interface ISiteExtensionManager
         /// <summary>
         /// Install or update a site extension
         /// </summary>
-        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer);
+        Task<SiteExtensionInfo> InstallExtension(string id, string version, string feedUrl, SiteExtensionInfo.SiteExtensionType type, ITracer tracer, string installationArgs);
```

Comment:
@SergeyKanzhelev the behavior if installationArgs = "a b" will be %1 = a and %2 = b